docs/html/reference/requirements-file-format.md

```diff
@@ -161,7 +160,7 @@ This disables the use of wheels (cached or otherwise). This could mean that buil
 This mechanism is only preserved for backwards compatibility and should be considered deprecated. A future release of pip may drop these options.
```

Comment:
After this PR, there's only one option left (`--global-option`). Although it might be worth it to document `--build-option` in this section.

Also, GitHub won't let me comment on the correct line, but there's a warning about disabling the use of wheels above, which should no longer be relevant.